staging/src/k8s.io/code-generator/cmd/client-gen/generators/client_generator.go

```diff
@@ -46,6 +46,18 @@ func NameSystems(pluralExceptions map[string]string) namer.NameSystems {
 			// you can put your fully qualified package like
 			// to generate a name that doesn't conflict with your group.
 			// "k8s.io/apis/events/v1beta1.Event": "EventResource"
+			"github.com/openshift/origin/pkg/build/apis/build/v1.Build":          "BuildResource",
```

Comment:
drop this patch. Not needed anymore for origin.

Reply:
Done.